components/job-orchestration/job_orchestration/job_config.py

```diff
@@ -31,4 +31,6 @@ class SearchConfig(BaseModel):
     search_controller_host: str
     search_controller_port: int
     wildcard_query: str
+    begin_timestamp: int = None
+    end_timestamp: int = None
```

Comment:
Technically, we can't assign `None` to a type `int` since `None` has it's own type (I'm sure `mypy` would complain). We should use `Optional[type_t]` instead. But since we're not using `mypy` yet I guess we can forget about it for now...?

Reply:
Fair enough. Yeah, it's not a big deal atm, but I may as well use the right type.